config/site.yml.tmpl

```diff
@@ -25,6 +25,11 @@ secure_cookies: false
 #
 secret_token: "change-me"
 
+# By default, tell Rails to not serve static assets. The best practice is to
```

Comment:
Is it the bit that says 'By default' that makes this bad? The functionality is to have Rails serve or not serve static assets. I'm a bit confused as to how the first line doesn't describe the functionality of the option.

Definitely need the reason for the default though. I'll add that in.